### PR TITLE
Use millis() instead of delay to set the execution period.

### PR DESCRIPTION
**Pull request by jlbirccyn**
Use millis() instead of delay to set the execution period.
Period moved as constant

### CODE REVIEW

File: src/arduino/respi_v0.cpp [hunk 1]
```diff
@@ -314,140 +316,152 @@ void loop() {
...
+        #else
+        lcd.print("c");
+        lcd.print(futureConsigneNbCycle);
+        lcd.print("/pl");
+        lcd.print(futureConsignePressionPlateauMax);
+        lcd.print("/pep");
+        lcd.print(futureConsignePressionPEP);
+        #endif
+      }
+  	}
```
Comment (dsferruzza) [inline src/arduino/respi_v0.cpp]: Il y a un soucis d'indentation ici, non ?

Comment (jlbirccyn) [reply src/arduino/respi_v0.cpp]: Oui, mélange tab / espaces à la ligne 464. Je rectifie.

**Comment by jlbirccyn** [comment]
C'est corrigé

**Comment by jabby** [comment]
Need a test with the arduino board. 
@dsferruzza are you at The Palace? If so, can you try with the oscilloscope and check with @waxzce if the problem is gone with this modification.

**Comment by dsferruzza** [comment]
> 
> 
> Need a test with the arduino board.
> @dsferruzza are you at The Palace? If so, can you try with the oscilloscope and check with @waxzce if the problem is gone with this modification.

I’m not likely to go out today, but:

1. We can try to test it remotely with someone there
2. Maybe @Mefl has the equipments/time to test it at home?

**Comment by jtanguy** [comment]
I am at Palace, I'll test

**Comment by jabby** [comment]
@Mefl @jtanguy @dsferruzza @jlbirccyn 
As we speak today, I will merge this PR.

**Pull request merged**